tmt-app/src/Components/MajorPage.jsx

```diff
@@ -1,18 +1,48 @@
-import { Container, Typography, Grid, Card } from "@mui/material";
+import {
+  Container,
+  Typography,
+  Grid,
+  Card,
+  List,
+  ListItemText,
+} from "@mui/material";
 import React from "react";
 import { useParams } from "react-router-dom";
 import { MAJORS_LIST } from "../utils/constants";
 import Breadcrumb from "./Breadcrumb";
 
 function MajorPage() {
   const params = useParams();
-  let data = [];
+  let data = MAJORS_LIST[`"${params.major}"`];
```

Comment:
Isn't params.major a string already? Template literal wrapping double quotes is probably still excessive

Reply:
I would get an error when I remove the quotes

Reply:
What's the error?

Reply:
![image](https://github.com/BobaTalks/pick-a-major/assets/36683390/f0366569-2f02-4e35-8f36-50181968aadf)